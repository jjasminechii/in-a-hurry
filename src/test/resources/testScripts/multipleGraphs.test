# Find a path with Dijkstra's algorithm for paths of
# two graphs

CreateGraph simpleGraph
CreateGraph secondGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode secondGraph C
AddNode secondGraph D

AddEdge simpleGraph A B 10.0
AddEdge secondGraph C D 1.0

FindPath simpleGraph A B

FindPath secondGraph C D