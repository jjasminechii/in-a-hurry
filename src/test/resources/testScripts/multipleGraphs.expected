# Find a path with Dijkstra's algorithm for paths of
# two graphs

created graph simpleGraph
created graph secondGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to secondGraph
added node D to secondGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from C to D in secondGraph

path from A to B:
A to B with weight 10.000
total cost: 10.000

path from C to D:
C to D with weight 1.000
total cost: 1.000

